src/wp-includes/class-wp-theme-json.php

```diff
@@ -2713,58 +2715,77 @@ private static function get_block_nodes( $theme_json, $selectors = array(), $opt
 		}
 
 		foreach ( $theme_json['styles']['blocks'] as $name => $node ) {
-			$selector = null;
-			if ( isset( $selectors[ $name ]['selector'] ) ) {
-				$selector = $selectors[ $name ]['selector'];
-			}
 
-			$duotone_selector = null;
-			if ( isset( $selectors[ $name ]['duotone'] ) ) {
-				$duotone_selector = $selectors[ $name ]['duotone'];
-			}
+			$include_node_paths_only = $options['include_node_paths_only'] ?? false;
+			$node_path = array( 'styles', 'blocks', $name );
+			if ( $include_node_paths_only ) {
+				$nodes[] = $node_path;
+			} else {
```

Comment:
Instead of using `if ... else ...`, can you use an early return here please? This helps code readability in general, but for changes like this it's especially useful since it means a lot fewer code lines are affected. This way you could leave most of the indentation below as it previously was.

Since this is a loop, instead of `return`, we'd need to use `continue` here.

Reply:
Done in https://github.com/WordPress/wordpress-develop/pull/7486/commits/dbbd01066088984f1a41ab24a83e58a46b9a27c4